frontend/src/app/footer/footer.component.html

```diff
@@ -9,7 +9,7 @@ <h3 class="nav-title">Informacje</h3>
             <a class="nav-link" href="#">Strona główna</a>
             <a class="nav-link" href="#">Załóż konto</a>
             <a class="nav-link" href="#">Kontakt z administratorem</a>
-            <a class="nav-link" href="#">O nas</a>
+            <a class="nav-link" href="about-us">O nas</a>
```

Comment:
`o-nas` intead of `about-us`

Reply:
+ You need to change routing on old Django-based footer (see for setting `ANGULAR_ROOT`)